library/ssl_tls.c

```diff
@@ -2644,7 +2644,7 @@ static int ssl_parse_certificate_verify( mbedtls_ssl_context *ssl,
     if( ssl->session_negotiate->verify_result != 0 )
     {
         MBEDTLS_SSL_DEBUG_MSG( 3, ( "! Certificate verification flags %x",
```

Comment:
Not strictly speaking in scope, but this is a 32-bit flag mask, so `%08x` would make more sense.